src/ImageSharp/Formats/Png/PngDecoderCore.cs

```diff
@@ -264,6 +329,14 @@ public ImageInfo Identify(BufferedReadStream stream, CancellationToken cancellat
                         case PngChunkType.Header:
                             this.ReadHeaderChunk(pngMetadata, chunk.Data.GetSpan());
                             break;
+                        case PngChunkType.AnimationControl:
+                            if (this.isSimplePng || this.animationControl is not null)
```

Comment:
I don't think we need the additional checks here. A standard png will not have this chunk.

Do you have examples of APNGs with multiple animation controls?